packages/breadcrumbs/src/elements/Breadcrumb.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Children, cloneElement, HTMLAttributes } from 'react';
+import { useBreadcrumb } from '@zendeskgarden/container-breadcrumb';
+
+import { StyledBreadcrumb } from '../styled/StyledBreadcrumb';
+import { StyledItem } from '../styled/StyledItem';
+
+export const Breadcrumb: React.FC<HTMLAttributes<HTMLOListElement>> = ({ children, ...props }) => {
```

Comment:
This should be using `React.forwardRef` to supply the `<nav>` ref to any consumers.